src/mame/drivers/pofo.cpp

```diff
@@ -1003,6 +852,7 @@ void portfolio_state::machine_start()
 void portfolio_state::machine_reset()
 {
 	m_lcdc->reset();
+	m_keyboard->reset();
```

Comment:
Subdevices are automatically reset.  Is there a reason for doing an extra reset?

Reply:
I assumed so as well, but then I saw the LCD controller was being reset here previously...
Looks like all of these resets here should just be removed since they are all subdevices anyway.

Reply:
Well, assuming there isn’t something really whacky going on, I’d assume the other subdevice resets should be safe to remove.